### PR TITLE
test(@toss/emotion-utils): Add FullHeight Component Test Code

### PR DESCRIPTION
**Pull request by ssi02014**
## Overview
I worked test code for the "FullHeight".
Please review this pull request to see if it's appropriate. 🙏

## References

- [Set Size of Window in Jest](https://stackoverflow.com/questions/60396600/set-size-of-window-in-jest-and-jest-dom-and-jsdom)

```ts
Object.defineProperty(window, 'innerWidth', { writable: true, configurable: true, value: 300 });
```

- [jsdom default Value (innerWidth: 1024, innerHeight: 768)](https://github.com/jsdom/jsdom/blob/0cba358253fd5530af0685ac48c2535992464d06/lib/jsdom/browser/Window.js#L587-L588)

스크린샷 2023-07-30 오전 2 54 09

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/react/emotion-utils/src/FullHeight.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const setInnerHeight = (height: number) => {
+  Object.defineProperty(window, 'innerHeight', { writable: true, configurable: true, value: height });
+};
+
+describe('FullHeight', () => {
+  it('should have height 768px', () => {
+    render(<FullHeight>FullHeight</FullHeight>);
+
+    const fullHeightBox = screen.getByText('FullHeight');
```
Comment (okinawaa) [inline packages/react/emotion-utils/src/FullHeight.spec.tsx:12]: how about seperate FullHeight text? 
It's not directly related to the test, but it's a means of testing.
and it can sustain SSOT

```tsx

const CHILDREN_TEXT = 'FullHeight'; // I'd like to change the variable name to something else


describe('FullHeight', () => {
  it('should have height 768px', () => {
    render(<FullHeight>{CHILDREN_TEXT}</FullHeight>);

    const fullHeightBox = screen.getByText(CHILDREN_TEXT);
```

**Review by okinawaa — CHANGES_REQUESTED** [review]
In the context of testing, the current `768` and `1024` belong to the same test group. So there is no need to test twice. 769,780,781,782 .. Same reason not to test etc.

File: packages/react/emotion-utils/src/FullHeight.spec.tsx (hunk 1 above)
Comment (ssi02014) [reply packages/react/emotion-utils/src/FullHeight.spec.tsx:12]: @okinawaa 
Thank you for your feedback.
Your comments resonate with us. I will fix the code.

**Comment by ssi02014** [comment]
@okinawaa 
>Because bug likes boundaries
Is there no need to test situations where height is 0 or negative?

The height of `FullHeight` depends on the internal `height` state, so the default value of height is `0`, so the case of 0 makes sense to consider. 

But is there a case where "window.innerHeight" could be `0` or `negative`? 

Actually, innerHeight is the `inner height value of the browser window`, which can get close to zero, but it's not likely to be 0 or negative.

So I don't think we need to deal with it. However, I'm open to adding a case of 0 if the majority thinks it's necessary.

(However, it would probably be code in much the same way as the 2nd test case).

File: packages/react/emotion-utils/src/FullHeight.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    const fullHeightBox = screen.getByText(CHILDREN_TEXT);
+
+    expect(window.innerHeight).toBe(768);
+
+    expect(fullHeightBox).toBeInTheDocument();
+    expect(fullHeightBox).toHaveStyle('height: 768px');
+  });
+
+  it('should have height 1024px', () => {
```
Comment (okinawaa) [inline packages/react/emotion-utils/src/FullHeight.spec.tsx:22]: Could you write two different test descriptions differently😄

One is to follow the default height, and the other is to follow the height that you set, 
so what test codes want to verify is different

Comment (ssi02014) [reply packages/react/emotion-utils/src/FullHeight.spec.tsx]: @okinawaa 
Thank you🙏 I fixed the code again.

**Review by okinawaa — APPROVED** [review]
cool thanks !

**Comment by ssi02014** [comment]
@okinawaa Thanks 🤗🎉
